tests/test_tensor/test_optimizers.py

```diff
@@ -214,3 +252,33 @@ def test_multiloss(backend, executor):
 
     if executor is not None:
         executor.shutdown()
+
+
+@pytest.mark.parametrize('backend', [jax_case, autograd_case,
+                                     tensorflow_case])
+def test_shared_tags(tagged_qaoa_tn,backend):
+    n,depth,psi0 = tagged_qaoa_tn
+
+    H = qu.ham_heis(n,j=(0.,0.,-1.),b=(1.,0.,0.),cyclic=True,)
```

Comment:
If possible, can you use pep8 style with spaces between arguments and operators etc.? Most editors can turn on such linting.

Reply:
Sure, of course! I ran pylama on `optimise.py` but I forgot to run it on the test file. I haven't been able to get the pylint sublime plugin working yet